app/src/pages/Play/GameRoundInstructionCard.tsx

```diff
@@ -8,6 +8,12 @@ export enum GameRound {
   Password = "Password"
 }
 
+export const GameRounds = [
+  GameRound.Taboo,
+  GameRound.Charades,
+  GameRound.Password
+]
```

Comment:
non trivial to enumerate the items in an enum, so this will need to be kept in sync with GameRound enum.